services/web-server/src/login/clients/GithubClient.js

```diff
@@ -1,18 +1,50 @@
+const assert = require('assert');
 const request = require('superagent');
 const Debug = require('debug');
 
 const debug = Debug('GithubClient');
 const baseUrl = 'https://api.github.com';
 
 module.exports = class GithubClient {
+  constructor({ accessToken }) {
+    assert(accessToken, 'An access token is required to access Github endpoints');
```

Comment:
This is different from the OAuth App's credentials.  So maybe we have to have *three* different sets of GitHub credentials?

Reply:
I think this is just the oauth access token we get after the user login with Github.